cfme/tests/containers/test_containers_properties.py

```diff
@@ -40,7 +41,8 @@ def test_pods_properties_rel(provider, rel):
     the fields in the Properties table
     """
     navigate_to(Pod, 'All')
-    ui_pods = [r.name.text for r in list_tbl_pods.rows()]
+    ui_pods = sample([r.name.text for r in list_tbl_pods.rows()],
```

Comment:
Oh yes, sorry for that, I had some problem to run the tests locally so I waited for PRT results. anyway - fixed..